source/_components/folder_watcher.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "folder watcher"
+description: "Component for monitoring changes within the filesystem."
+date: 2018-03-11 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: System Monitor
+ha_iot_class: "Local Polling"
+ha_release: 0.67
+---
+
+This component adds [Watchdog](https://pythonhosted.org/watchdog/) file system monitoring, publishing events on the Home-Assistant bus on the creation/deletion/modification of files.
```

Comment:
Please change **Home-Assistant** to **Home Assistant**.

Reply:
ok